.buildkite/hooks/pre-exit

```diff
@@ -17,7 +17,5 @@ cleanup
 unset_secrets
 
 # integrations-parallel-aws and intregrations-parallel-aws_logs
-unset ELASTIC_PACKAGE_AWS_ACCESS_KEY
-unset ELASTIC_PACKAGE_AWS_SECRET_KEY
 unset AWS_ACCESS_KEY_ID
 unset AWS_SECRET_ACCESS_KEY
```

Comment:
I guess this comment should be just for the cloud-cleanup step now ? 
In elastic-package, there is no other usage of creating AWS resources via Terraform.

Or could those two commands to unset AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY be removed too now ? IIRC those two variables were defined/exported just in `.buildkite/scripts/cloud-cleanup.sh`

Reply:
Let me delete those two variables and the comment too